app/config/services/services.xml

```diff
@@ -72,7 +69,6 @@
 
 		<!-- Security -->
 		<prototype namespace="AppBundle\Security\Voter\" resource="../../../src/Security/Voter" />
-		<service id="AppBundle\Security\FailedLoginAttemptFactory"/>
```

Comment:
Hmm, I though I already did fix that file.

Reply:
auth2 is not up to date so it's possible